src/config.h.in

```diff
@@ -9,7 +9,7 @@
 #define TIMESCALEDB_MOD_VERSION "@VERSION_MOD@"
 #define BUILD_OS_NAME "@CMAKE_SYSTEM_NAME@"
 #define BUILD_OS_VERSION "@CMAKE_SYSTEM_VERSION@"
-#define PG_VERSION "@PG_VERSION@"
+#define CANONICAL_PG_VERSION "@PG_VERSION@"
```

Comment:
Oh, I thought you were going to delete this? I'm ok with just going with the Postgres-defined `PG_VERSION`.

Reply:
I thought about it and I think it is probably better for us to have a canonical form so that `10.5` and `10.5 Ubuntu 16.04...` don't show up separately in telemetry

Reply:
I'd prefer not to add this (redundant) variable in TS if possible. It looks like if you configure Postgres with the `--with-extra-version=<string>` flag, you add extra crap at the end of `PG_VERSION`. Do we control the way Ubuntu is configured / built? Can you check if this flag is set and if so, unset it?

Reply:
Actually, I think `PACKAGE_VERSION` is what we're looking for.

Reply:
I do not get to control anything about PostgreSQL is built/compiled on Ubuntu.

I'll look into PACKAGE_VERSION

Reply:
Okay, yes I think you are right, so I updated to use `PACKAGE_VERSION`